using a in 测试.错误处理.函数内不识变量

a()
/*
 😰 请先定义'b'再使用
“测试/错误处理/函数内不识变量.ul”第2行：print(b)
调用层级如下
见第3行：a()
*/